### PR TITLE
feat(LyricsCommand): Get lyrics from current playing song

### PR DESCRIPTION
**Pull request by lxndr-rl**
Show Lyrics & albumArt from current playing song
When the number of characters exceeds the limit is divided into several embeds.

image

### CODE REVIEW

File: src/commands/LyricsCommand.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import { IMessage } from "../../typings";
+import { DefineCommand } from "../utils/decorators/DefineCommand";
+import { createEmbed } from "../utils/createEmbed";
+import { isUserInTheVoiceChannel, isSameVoiceChannel, isValidVoiceChannel } from "../utils/decorators/MusicHelper";
+import fetch from "node-fetch";
+
+@DefineCommand({
+    aliases: ["ly"],
+    name: "lyrics",
+    description: "Get lyrics from the current song",
```
Comment (mzrtamp) [inline src/commands/LyricsCommand.ts]:
```suggestion
    description: "Get lyrics from the current music",
```

File: src/commands/LyricsCommand.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                        cantEmbeds = 1;
+                    } else {
+                        for (let i = 2; i < 10; i++) {
+                            if (charLength < 2048 * i) {
+                                cantEmbeds = i;
+                                break;
+                            }
+                        }
+                    }
+                    if (!album_art) album_art = "https://kknd26.ru/images/no_photo.png";
```
Comment (mzrtamp) [inline src/commands/LyricsCommand.ts]: Use Zhycorp's logo instead for no photo
```suggestion
                    if (!album_art) album_art = "https://api.zhycorp.com/assets/images/logo.png";
```

**Review by mzrtamp — CHANGES_REQUESTED** [review]

**Comment by mzrtamp** [comment]
Idk about this, please help.
> npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.

**Review by mzrtamp — APPROVED** [review]
Thanks for your contribution!

**Comment by lxndr-rl** [comment]
> cipm can only install packages with an existing package-lock.json

It seems that "Checks" does not generate the `package-lock.json` file by itself (this is not an error if the installation is done on the device `npm install` -> `npm start`)

**Comment by mzrtamp** [comment]
> It seems that "Checks" does not generate the `package-lock.json` file by itself (this is not an error if the installation is done on the device `npm start`)

It's k, thanks

**Pull request merged**